docs/source/guide/migrations3.rst

```diff
@@ -0,0 +1,160 @@
+.. _guide_migration_s3:
+
+Amazon S3
+=========
+Boto 2.x contains a number of customizations to make working with Amazon S3 buckets and keys easy.
+
+Creating the Connection
+-----------------------
+Boto 3 has both low-level clients and higher-level resources. For Amazon S3, the higher-level resources are the most similar to Boto 2.x's ``s3`` module::
+
+    # Boto 2.x
+    import boto
+    s3_connection = boto.connect_s3()
+
+    # Boto 3
+    import boto3
+    s3 = boto3.resource('s3')
+
+Creating a Bucket
+-----------------
+Creating a bucket in Boto 2 and Boto 3 is very similar, except that in Boto 3 all action parameters must be passed via keyword arguments and a bucket configuration must be specified manually::
+
+    # Boto 2.x
+    s3_connection.create_bucket('mybucket')
+    s3_connection.create_bucket('mybucket', location=Location.USWest)
+
+    # Boto 3
+    s3.create_bucket(Bucket='mybucket')
+    s3.create_bucket(Bucket='mybucket', CreateBucketConfiguration={
+        'Location': 'us-west-1'})
```

Comment:
Thanks! The new docs in #77 will help prevent this sort of mixup in the future I hope.